### PR TITLE
ZH-trans: add generated/...

### PR DESCRIPTION
**Pull request by xichengliudui**
ZH-trans: add generated/...

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+<!--
+Renews certificates for a Kubernetes cluster
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md]: Renews 有续订的意思，译成 更新 可能不太好..更新应该是 update

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">renew 的帮助信息</td>
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md]: 英文注释可以插在这里

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+<!--
+### Options
+-->
+
+### 选项
+
+<!--
+<td></td><td style="line-height: 130%; word-wrap: break-word;">help for renew</td>
+-->
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew.md]: 不必把这样原文单独提出来..打破了表格结构

**Comment by chenrui333** [comment]
looks like the base branch is not originally from release-1.12, a new PR might be required

**Comment by Hartmarken348** [comment]
Add blue general

**Pull request merged**